exercises/target/canonical-data.json

```diff
@@ -0,0 +1,65 @@
+{
+  "exercise": "target",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "Retun the correct amount earnt by a dart landing in a given point in the target problem.",
```

Comment:
While not wrong, `earnt` is not the common spelling, `earned` is more the more common spelling.